core/database/foxx/api/schema_router.js

```diff
@@ -515,7 +660,30 @@ router
             fixSchOwnNm(sch);
 
             res.send([sch]);
+            logger.logRequestSuccess({
+                client: req.queryParams?.client,
+                correlationId: req.headers["x-correlation-id"],
+                httpVerb: "GET",
+                routePath: basePath + "/view",
+                status: "Success",
+                description: `View schema. ID: ${req.queryParams.id}`,
+                extra: {
+                    own_id: sch.own_id,
+                    own_nm: sch.own_nm,
+                    id: sch.id,
+                    desc: sch.desc,
```

Comment:
This will print too much information, we don't want to include the desc. Or if we do, limit the char output. Include pub, sys attributes.